tests/main/change-errors/task.yaml

```diff
@@ -0,0 +1,7 @@
+summary: Checks for cli errors of the change command.
+
+execute: |
+  echo "When an invalid ID is given to the change command it shows an error"
+  if snap change 10000000; then
+    echo "Expected error when trying change on invalid ID" && exit 1
```

Comment:
The `&& exit 1` here would nicer as `\nexit 1`.. trivial and not a blocker, though. Merging..